File: server/php/src/Infrastructure/Mappers/PaymentDetails/CardFraudResultsMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\CardFraudResults as CardFraudResultsDto;
use OnlinePayments\Sdk\Domain\CardFraudResults as CardFraudResultsSdk;

final class CardFraudResultsMapper
{
    public static function mapFromSdkResponse(?CardFraudResultsSdk $response): ?CardFraudResultsDto
    {
        if ($response === null) {
            return null;
        }

        $dto = new CardFraudResultsDto();
        $dto->avsResult = $response?->avsResult ?? null;
        $dto->fraudServiceResult = $response?->fraudServiceResult ?? null;
        $dto->cvvResult = $response?->cvvResult ?? null;

        return $dto;
    }
}

File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/CardFraudResultsMapper.cs
using CardFraudResultsSdk = OnlinePayments.Sdk.Domain.CardFraudResults;
using CardFraudResultsDto = Business.Domain.Payments.PaymentDetails.CardFraudResults;

namespace Infrastructure.Mappers.PaymentDetails;

public static class CardFraudResultsMapper
{
    public static CardFraudResultsDto Map(CardFraudResultsSdk? response)
    {
        return new()
        {
            AvsResult = response?.AvsResult,
            FraudServiceResult = response?.FraudServiceResult,
            CvvResult = response?.CvvResult
        };
    }
}